build.gradle

```diff
@@ -242,7 +244,27 @@ subprojects {
       }
     }
 
```

Comment:
Btw pls put the instruction for handling logs in IntelliJ posted on the PR itself into README.md as well